core/src/Cabana_ParticleInit.hpp

```diff
@@ -134,8 +136,8 @@ int createParticles(
         }
     };
 
-    Kokkos::RangePolicy<ExecutionSpace> exec_policy( exec_space, 0,
-                                                     num_particles );
+    Kokkos::RangePolicy<ExecutionSpace> exec_policy(
+        exec_space, 0, previous_num_particles + num_particles );
```

Comment:
This should be ` exec_space, previous_num_particles, previous_num_particles + num_particles`, right?